tools/ide_plugins/pysa-vscode/package.json

```diff
@@ -38,5 +38,34 @@
     "tslint-microsoft-contrib": "^5.0.1",
     "typescript": "^3.5.1",
     "vscode": "^1.1.7"
+  },
+  "categories": [
+    "Programming Languages"
+  ],
+  "contributes": {
+      "languages": [{
+          "id": "pysa",
+          "aliases": ["Pysa", "pysa"],
+          "extensions": [".pysa"],
+          "configuration": "./language-configuration.json"
+      },
+      {
+          "id": "taint",
+          "aliases": ["Pysa", "taint"],
+          "extensions": ["taint.config"],
+          "configuration": "./language-configuration.json"
+      }
+      ],
+      "grammars": [{
+          "language": "pysa",
+          "scopeName": "source.pysa",
+          "path": "./syntaxes/Pysa.tmLanguage"
+      },
+      {
+          "language": "taint",
+          "scopeName": "taint.config",
```

Comment:
Naming is inconsistent between these two languages. Should this be `source.taint`?

Reply:
I changed both scope names to `source.pysa.model` and `source.pysa.config` to keep it consistent with the language-naming - what do you think?